lib/wallaby/browser.ex

```diff
@@ -980,6 +980,63 @@ defmodule Wallaby.Browser do
     current_url(session) == "about:blank"
   end
 
+  @doc """
+  Accepts all subsequent JavaScript dialogs in the given session.
+  """
+  def accept_dialogs(%Session{}=session) do
+    Driver.accept_dialogs(session)
+    session
+  end
+
+  @doc """
+  Dismisses all subsequent JavaScript dialogs in the given session.
+  """
+  def dismiss_dialogs(%Session{}=session) do
+    Driver.dismiss_dialogs(session)
+    session
+  end
+
+  @doc """
+  Accepts one alert dialog, which is triggered within the specified `fun`.
+  Returns the last message that was presented to the user.
```

Comment:
at least for the first one of those it might be nice to show a code example of what might be meant by `triggered by specified fun` :)